setup.cfg

```diff
@@ -39,6 +39,7 @@ ignore_errors = True
 # tox test environments for generating licenses and testing optional dependencies
 [tox:tox]
 envlist=
+    validate
```

Comment:
What is this for?

Reply:
Ah, I was using a tox environment to validate some of the saving stuff but it proved unessesery. I've removed it now.